### ISSUE TITLE
Adjust tabIndex for main modal component

### ISSUE THREAD

**Issue by jakemobilize** [reactjs/react-modal#902]
### Summary:
Currently the tabIndex in the ModalPortal.js file cannot be altered - always set to -1. When I have a modal and then run an accessibility testing tool (ANDI) I get an alert that the ReactModal__Content element is focusable but not tabbable.

### Steps to reproduce:

 1. Create a modal with tabbable content inside
 2. Download [ANDI](https://www.ssa.gov/accessibility/andi/help/install.html) and follow bookmarking instructions
 3. Navigate to the page with your modal and run ANDI (click bookmark)
 4. Open modal and then refresh ANDI to run it again with the open modal
 5. You should get the following accessibility alert:
 ```
Keyboard Access Alerts: (1)
1. Focusable element is not in keyboard tab order; should it be tabbable?
```
6. By clicking on this error, it will highlight the element in question and reveal that the tabindex is set to -1.

### Expected behavior:
Set tabIndex as a prop in the Modal component to be able to customize this. OR have the ability to make the react modal component non-focusable--using a number of aria attributes for this did not work and I didn't see anything listed in current props that would allow for this.

### Additional Notes:
All tabbing inside the modal works as expected, it is simply the modal component itself
